SSA/Projects/InstCombine/Base.lean

```diff
@@ -88,6 +88,22 @@ deriving Repr, DecidableEq, Inhabited
 
 /-- Homogeneous, binary operations -/
 inductive MOp.BinaryOp : Type
+  | and
+  | or
+  | xor
+  | shl
+  | lshr
+  | ashr
+  | urem
+  | srem
+  | add (nswnuw  : Bool × Bool := ⟨ false , false ⟩ )
+  | mul
+  | sub
+  | sdiv
+  | udiv
+deriving DecidableEq, Inhabited
+/-- Homogeneous, binary operations -/
+inductive g : Type
```

Comment:
Leave an empty line in between definition/inductives (there are some exceptions, when items are small and very closely related, but 99/100 times you want the empty line)